awsagentprovider/src/main/java/software/amazon/opentelemetry/javaagent/providers/AwsApplicationSignalsCustomizerProvider.java

```diff
@@ -207,8 +207,25 @@ private SdkTracerProviderBuilder customizeTracerProviderBuilder(
       // If running on Lambda, we just need to export 100% spans and skip generating any Application
       // Signals metrics.
       if (isLambdaEnvironment()) {
+        String tracesEndpoint =
+            Optional.ofNullable(System.getenv(AWS_XRAY_DAEMON_ADDRESS_CONFIG))
+                .orElse(DEFAULT_UDP_ENDPOINT);
+        SpanExporter spanExporter =
+            new OtlpUdpSpanExporterBuilder()
+                .setPayloadSampleDecision(TracePayloadSampleDecision.UNSAMPLED)
+                .setEndpoint(tracesEndpoint)
+                .build();
+
+        // Wrap the udp exporter with the AwsMetricsAttributesSpanExporter to add Application
```

Comment:
This is important. Good catch.

Reply:
In Python and NodeJS, if customer has set the customized OTLP endpoint through Env Var, we will not enable UDP exporter to send unsampled Span to the custom endpoint. Should we do the same for Java and .Net

JS: https://github.com/mxiamxia/aws-otel-js-instrumentation/blob/1bf42f613a3bc2142e396c5fabaef4f5245e0495/aws-distro-opentelemetry-node-autoinstrumentation/src/aws-opentelemetry-configurator.ts#L441C10-L441C36

Python: https://github.com/aws-observability/aws-otel-python-instrumentation/blob/main/aws-opentelemetry-distro/src/amazon/opentelemetry/distro/aws_opentelemetry_configurator.py#L314

Reply:
Yes. good callout. I will add that condition.